vizro-core/src/vizro/_themes/_palettes.py

```diff
@@ -0,0 +1,183 @@
+"""Color palettes by Vizro."""
+
+from types import SimpleNamespace
+
+from ._colors import colors
+
+qualitative_10 = [
```

Comment:
I'd call this just `qualitative` unless you think in future we're likely to have different palettes for `qualitative_2`, `qualitative_3` etc. that aren't just the first 2/3 entries of this palette.

Even if we do have those, probably it's useful to have a "standard default" `qualitiative` palette anyway, so I'd still just call this `qualitative`.